packages/angular/src/generators/application/application.spec.ts

```diff
@@ -1039,22 +1039,17 @@ describe('app', () => {
     });
   });
 
-  it('should generate correct main.ts', async () => {
```

Comment:
Is there a reason this test has been deleted?

Reply:
Good catch. No idea how that happened. I'll have a look and replace it.

Reply:
Added back